sim/common/wotlk/enchant_effects.go

```diff
@@ -269,7 +269,7 @@ func init() {
 		w.BaseDamageMax += 15
 	})
 
-	core.NewItemEffect(54998, func(agent core.Agent) {
+	core.NewItemEffect(41091, func(agent core.Agent) {
```

Comment:
This ID needs to match the value here: https://github.com/wowsims/wotlk/blob/8e734157beac1a64c1eb08b991d3ec39ea4f1c4e/sim/core/items/all_enchants.go#L104

So this would break the effect.

Also, seems like the ID was already correct to me: https://www.wowhead.com/wotlk/spell=54998/hand-mounted-pyro-rocket